src/modules/emote_menu/components/EmoteMenu.jsx

```diff
@@ -56,6 +68,9 @@ export default function EmoteMenu({toggleWhisper, appendToChat, onSetTip}) {
     function buttonPressCallback(event) {
       setAltPressed(event.altKey);
       setShiftPressed(event.shiftKey);
+      if (!event.shiftKey) {
+        cacheOrProcessEmoteClicks(null, false);
```

Comment:
if the popout is closed via other means (clicking out of the popout or closing it via the X in the upper right), these track events are lost. this operation may need to be in the parent. since `appendToChat` is a parent function, perhaps the track could be moved there instead.